java/core/src/main/java/com/microsoft/bond/protocol/ProtocolWriter.java

```diff
@@ -13,6 +13,12 @@
 public interface ProtocolWriter {
 
     /**
+     * @return true if bonded fields in this protocol are marshaled; false if
+     * they are serialized in the same protocol
+     */
+    boolean usesMarshaledBonded();
```

Comment:
I don't understand why this method is needed here. The .NET version doesn't have it.